nltk/tokenize/casual.py

```diff
@@ -350,17 +334,14 @@ def tokenize(self, text):
         # Normalize word lengthening
         if self.reduce_len:
             text = reduce_lengthening(text)
-        # Enable recognising phone numbers
-        if self.phone_number_regex:
-            global WORD_RE
-            WORD_RE = regex.compile(
-                r"""(%s)""" % "|".join(REGEXPS_PHONE),
-                regex.VERBOSE | regex.I | regex.UNICODE,
-            )
         # Shorten problematic sequences of characters
         safe_text = HANG_RE.sub(r"\1\1\1", text)
         # Tokenize:
         words = WORD_RE.findall(safe_text)
```

Comment:
This still doesn't seem quite right - if `self.phone_number_regex == True`, then `words = WORD_RE.findall(safe_text)` is ran, and then this `words` is overridden. So the `words = WORD_RE.findall(safe_text)` was not necessary if `self.phone_number_regex == True`.